src/AnalyzeRegistry.jl

```diff
@@ -113,6 +120,10 @@ function Base.show(io::IO, p::Package)
         else
             body *= "  * has continuous integration: false\n"
         end
+        if !isempty(p.lines_of_code)
+          body *= "  * lines of Julia code in `src`: $(count_julia_loc(p.lines_of_code, :src))\n"
+          body *= "  * lines of Julia code in `test`: $(count_julia_loc(p.lines_of_code, :test))\n"
+        end
```

Comment:
Also here, I think the number of lines is more juicy than CI or documentation, I would move these lines above :slightly_smiling_face:

Reply:
sounds good! how does this look?
```julia
julia> analyze(pkgdir(AnalyzeRegistry))
Package AnalyzeRegistry:
  * repo: 
  * uuid: e713c705-17e4-4cec-abe0-95bf5bf3e10c
  * is reachable: true
  * lines of Julia code in `src`: 326
  * lines of Julia code in `test`: 58
  * has license(s) in file: MIT
    * filename: LICENSE
    * OSI approved: true
  * has documentation: false
  * has tests: true
  * has continuous integration: true
    * GitHub Actions
```

Reply:
Looks good!